docs/introduction/Latus-Consensus.rst

```diff
@@ -11,12 +11,17 @@ In Latus, the chain is split into “consensus epochs”, where each epoch compr
 Slot leaders of a particular consensus epoch are chosen randomly before the epoch begins from the set of all sidechain forging stakeholders. The forging stake is a subset of all the coins managed by a sidechain. In fact each sidechain participant who wants to be a Forger must have some forging stake - i.e. a set of “ForgerBoxes” assigned to him. ForgerBox is a particular kind of Box that contains an amount of coins locked for forging, and some specific data used by the forger to prove its block-producing eligibility associated with that stake amount. The total amount of coins staked in ForgerBoxes is the total Forging Stake amount.
 The possibility of being a slot leader increases with the percentage of forging stake owned. It's possible to have more than one slot leader per slot. If more than one block is propagated, only one will be accepted by each node; the consensus rules will make sure that conflicting chains will eventually converge to a winning chain. Conversely, a consensus epoch could have empty slots if their slot leader (or leaders) have not created and propagated blocks for them.
 
+Forger Stake can be delegated to another node with makeForgerStake command. This command contain a list of transactions, each of them specifies publicKey of a sender, blockSignPublicKey and vrfPubKey of receiving node and coin amount.
+Also Forger Stake can be delegated by spendForgingStake command. In this case Forger Stake coins must be specified in forgerOutputs, coins specified in regularOutputs section will be moved fomr Forger Stakes and can be used as regular coins.
```

Comment:
Also, Forger Stake can be delegated by spendForgingStake command. In this case, Forger Stake coins must be specified in forgerOutputs, coins specified in regularOutputs section will be moved from Forger Stakes and can be used as regular coins.